web/themes/custom/itkdev/itkdev_project_theme/css/project-styles.css

```diff
@@ -4,6 +4,71 @@
     --theme-primary: #fff;
     --theme-secondary: #F4F4F5;
     --theme-brand: #047857;
-    --theme-brand-hover: #064E3B; 
+    --theme-brand-hover: #064E3B;
     --theme-border: #E4E4E7;
-}
\ No newline at end of file
+}
+
+.tablesort {
+  &.desc {
+    height: 100%;
+    vertical-align: text-bottom;
+  }
+
+  &.asc {
+    height: 100%;
+    vertical-align: sub;
+  }
+}
+
+/**
+ * @file
+ *
+ * Replaces standard throbber from stable9 with Olivero's.
+ *
+ * Copied and edited from:
+ * core/themes/stable9/css/system/components/autocomplete-loading.module.css
+ *
+ */
```

Comment:
Are you copying and editing from somewhere (Olivero) that has already copied and edited from another place (stable9)?